test/replication.test.js

```diff
@@ -1804,3 +1804,105 @@ describe('Replication / Change APIs', function() {
     return getPropValue(list, 'id');
   }
 });
+
+describe('Replication / Change APIs with multi tenancy', function() {
+  this.timeout(10000);
+  var dataSource, SourceModelWithTenant, TargetModelWithTenant;
+  var useSinceFilter;
+  var tid = 0; // per-test unique id used e.g. to build unique model names
+
+  beforeEach(function() {
+    tid++;
+    useSinceFilter = false;
+    var test = this;
+
+    dataSource = this.dataSource = loopback.createDataSource({
+      connector: loopback.Memory
+    });
+    SourceModelWithTenant = this.SourceModelWithTenant = PersistedModel.extend(
```

Comment:
It would be better to move all the tests to this format. I would rather do it separate to this pull request.